src/components/searchBox.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component, Fragment } from "react";
+import InputText from "./inputText";
+
+class SearchBox extends Component {
+    state = {};
+    render() {
+        return (
+            <Fragment>
+                <InputText label="What Pokemon are you looking for?" />
```

Comment:
Be careful, InputText is a component from another branch. If a checkout to this branch I wouldn't be able to run the application. If you merge this branch before POK-10 you are going to break the master branch